doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
+Dependent resources *within the same namespace as the owning CR* are
+tracked with the `ownerReference` field.
+
+`ownerReference` structure:
+  * apiVersion: {group}/{version}
+  * kind: {kind}
+  * name: {metadata.name}
+  * uid: {metadata.uid}
+
+**Example:**
+
+```yaml
+
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+An `annotation` is used instead of an `ownerReference` if the dependent
+resource is in a different namespace than the CR or the dependent
+resource is a cluster level resource.
+
+`annotation` structure:
+  * operator-sdk/primary-resource: {metadata.namepace}/{metadata.name}
+  * operator-sdk/primary-resource-type: {kind}.{group}
+
+*Note: <group> must be determined by splitting the apiVersion field at the "/"*
```

Comment:
I think would be easier to add here where the user can find the group of a primary-resource and we could use the samples to make it easier. Following my suggestion.

```suggestion
**NOTE**: The <group> can be found in the `apiVersion` metadata in the CR of the resource. See [here](https://github.com/operator-framework/operator-sdk-samples/blob/master/ansible/memcached-operator/deploy/crds/cache_v1alpha1_memcached_cr.yaml#L1) and example. Also, see that the <group> value is the `apiVersion` without the version as for example `cache.example.com`*
```